app/presenters/publishing_api/document_collection_presenter.rb

```diff
@@ -33,7 +33,7 @@ def links
       links = LinksPresenter.new(item).extract(
         %i(organisations policy_areas topics related_policies parent)
       )
-      links.merge!(documents: item.documents.map(&:content_id))
+      links.merge!(documents: item.documents.map(&:content_id).uniq)
```

Comment:
It looks like the unit test for this stubs documents and returns an array of OpenStructs which isn't to say it can't be refactored to work with `pluck`. I'll do this in a subsequent PR.